h5netcdf/core.py

```diff
@@ -445,15 +493,19 @@ def attrs(self):
     _cls_name = 'h5netcdf.Group'
 
     def _repr_body(self):
-        return (['Dimensions:'] +
-                ['    %s: %s' % (k, v) for k, v in self.dimensions.items()] +
-                ['Groups:'] +
-                ['    %s' % g for g in self.groups] +
-                ['Variables:'] +
-                ['    %s: %r %s' % (k, v.dimensions, v.dtype)
-                 for k, v in self.variables.items()] +
-                ['Attributes:'] +
-                ['    %s: %r' % (k, v) for k, v in self.attrs.items()])
+        return (
+            ['Dimensions:'] +
+            ['    %s: %s' % (k, ("Unlimited (current: %s)" %
+                                 self._current_dim_sizes[k])
+                             if v is None else v)
```

Comment:
indent three spaces to match `(`, not `k`

Reply:
This is the only indentation `flake8` accepts and in most parts of your code you adhere to it. I'll break it in a different way while still being `flake8` compliant so it looks a bit nicer.